yt/visualization/volume_rendering/scene.py

```diff
@@ -226,8 +226,25 @@ def render(self, camera=None):
         self._last_render = bmp
         return bmp
 
+    def _check_render(self, fname, render=True):    
+        # checks for existing render before saving,  in most cases we want to 
+        # render every time, but in some cases pulling the previous render is 
+        # desirable (e.g., if only changing sigma_clip or
+        # saving after a call to sc.show()).
+        if self._last_render is None:
+            mylog.warning("No previous rendered image found, rendering now and saving to %s", fname)
+            render = True
+        elif render:
+            mylog.info("Overwriting previous rendered image with new rendering, saving to %s", fname)
```

Comment:
OK -- good point, I added it because I thought the "saving to fname" that gets  logged was repetitive, but ya, if the call to render goes wrong it's helpful to have a separate message!

Reply:
Ok, added back the separate Save message log